shared/scss/abstracts/_variables.scss

```diff
@@ -6,3 +6,73 @@ $fontWeightRegular: 400;
 $fontWeightMedium: 500;
 $fontWeightSemibold: 600;
 $fontWeightBold: 700;
+
+$fontSizeXs: 0.75em;
+$fontSizeSm: 0.875em;
+$fontSizeBase: 1em;
+$fontSizeLg: 1.125em;
+$fontSizeXl: 1.25em;
+$fontSize2Xl: 1.5em;
+$fontSize3Xl: 1.875em;
+$fontSize4Xl: 2.25em;
+// To be continued...
+
+/*------------------------------------*\
+  #BREAKPOINTS
+\*------------------------------------*/
+
+$led4k: 160em;
+$alpha: 87.5em;
+$preAlpha: 75.0625em;
+$beta: 75em;
+$preBeta: 64em;
+$gamma: 62em;
+$lota: 31.25em;
+$preZeta: 47.9375em;
+$eta: 40em;
+$kappa: 25em;
+
+/*------------------------------------*\
+  #SPACES (padding & margin & gap) 
+\*------------------------------------*/
+$space1: calc(0.25em * 1);
+$space2: calc(0.25em * 2);
+$space3: calc(0.25em * 3);
+$space4: calc(0.25em * 4);
+$space5: calc(0.25em * 5);
+$space6: calc(0.25em * 6);
+$space7: calc(0.25em * 7);
+$space8: calc(0.25em * 8);
+$space9: calc(0.25em * 9);
+$space10: calc(0.25em * 10);
+$space11: calc(0.25em * 11);
+$space12: calc(0.25em * 12);
+// To be continued...
+
+/*------------------------------------*\
+  #TRANSITION DURATION
+\*------------------------------------*/
+
+$duration75: 75ms;
+$duration100: 100ms;
+$duration150: 150ms;
+$duration200: 200ms;
+$duration300: 300ms;
+$duration500: 500ms;
+$duration700: 700ms;
+$duration1000: 1000ms;
```

Comment:
The same as before, try to use the features in Sass, like loop

Reply:
For these ones we can't use a for loop to generate them because they don't follow a consistant pattern like spaces variables but instead we can use a map to make it a little more clean

Reply:
```
$durations: 75, 100, 150, 200, 300, 500, 700, 1000;

@each $duration in $durations {
  $duration#{$duration}: #{$duration}ms;
}

```